(DOCSP-24141): Add docs release notes for w/e August 5 (#2082)
mongodb/docs-realm: docs-release-notes.md

# August 5, 2022

## .NET SDK

> - Usage Examples/React to Changes: New section w/tested code examples for "Check if the Collection Has Been Cleared"

Reviewer: ```suggestion
- Usage Examples/React to Changes: Added a new section with tested code examples for "Check if the Collection Has Been Cleared"
```

Reviewer: Should this use a nested list format like the other SDKs even though it's only one bullet?

## Node.js SDK
- Usage Examples

>   - Open & Close a Realm: Add note about `WriteCopy` only supporting Partition-Based Sync

Reviewer: ```suggestion
  - Open & Close a Realm: Add a note about `WriteCopy` only supporting Partition-Based Sync
```

  - Flexible Sync: Remove `reRunOnOpen` API from docs

## React Native SDK
- Usage Examples

>   - Open & Close a Realm: Add note about `WriteCopy` only supporting Partition-Based Sync

Reviewer: ```suggestion
  - Open & Close a Realm: Add a note about `WriteCopy` only supporting Partition-Based Sync
```
